scripts/commands/wizard/quiz.js

```diff
@@ -0,0 +1,175 @@
+    {
+      answer: responses.correctAnswer,
+      correct: true,
+      feedback: responses.correctFeedback
+    }
+  ]
+  log.info(`Question: "${question}"`)
+  log.info(`Answer (Correct): "${choices[0].answer}"`)
+  log.info(`Feedback: "${choices[0].feedback}"`)
+
+  let askAgain = true
+  let wrongAnswer
+
+  while (askAgain) {
+    // repeat as many times as they want to create new wrong answers
+    responses = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'incorrectAnswer',
+        message: 'What incorrect option should be presented to the learner?',
+        validate: validateStringPresent
+      },
+      {
+        type: 'input',
+        name: 'incorrectFeedback',
+        message: 'What failure message should the learner see if they pick this option? Be sure to explain why this choice is wrong so they can feel more confident when choosing again.',
+        validate: validateStringPresent
+      }
+    ])
+
+    wrongAnswer = {
+      answer: responses.incorrectAnswer,
+      correct: false,
+      feedback: responses.incorrectFeedback
+    }
+
+    log.info(`Answer (Incorrect): "${wrongAnswer.answer}"`)
+    log.info(`Feedback: "${wrongAnswer.feedback}"`)
+
+    choices.push(wrongAnswer)
+
+    log.info(`You currently have 1 correct answer and ${choices.length - 1} wrong answer${choices.length > 2 ? 's' : ''}. (I recommend providing 2-3 incorrect options.)`)
+
+    if (!(await promptRepeat('wrong answer'))) {
+      askAgain = false
+    }
+  } // end while loop
+
+  // shuffle right and wrong answers to change position of correct one
+  for (let i = (choices.length - 1); i > 0; i--) {
+    const j = Math.floor(Math.random() * i)
+    const temp = choices[i]
+    choices[i] = choices[j]
+    choices[j] = temp
+  }
+
+  logList(`Cool! Here's what we get when we mix up the order of the answer choices`, choices.map((choice, index) => `Option ${index + 1}: \n    ‣ Answer (${choice.correct ? 'Correct' : 'Incorrect'}): ${choice.answer} \n    ‣ Feedback: ${choice.feedback}`))
+  await api.lessons.updateQuiz(tutorial, lesson, { question, choices })
+  await afterQuizCreate(tutorial, lesson, { createLesson, createTutorial, createResource })
+} // end createQuiz
+
+// *** TRANSITIONAL DIALOGS & PROMPTS ***
+
+async function createQuizIntro ({ createLesson, createTutorial, createResource }) {
+  log.info("Let's create a multiple-choice quiz! This will only work if you've already created the tutorial and lesson files.")
+
+  if (await promptFilesReady()) {
+    const tutorial = await selectTutorial('quiz', { createTutorial, createResource, createLesson, createQuiz })
+    const lesson = await selectLesson(tutorial)
+
+    if (lesson) { // skip if value is null because there weren't multiple choice lessons in tutorial
+      logList(`Great! You've chosen to build a quiz for`, [`Tutorial: ${tutorial.title}`, `Lesson: ${lesson.title}`])
+
+      const overwrite = api.lessons.isQuizPristine(tutorial, lesson)
```

Comment:
Just a nick-pick. Using `should*` indicates that it is boolean and that we will perform the override or not depending on its value. 

```suggestion
      const shouldOverride = api.lessons.isQuizPristine(tutorial, lesson)
```